harbor_exporter.go

```diff
@@ -126,6 +127,7 @@ func createMetrics(instanceName string) {
 	allMetrics["system_with_chartmuseum"] = newMetricInfo(instanceName, "system_with_chartmuseum", "If harbor has chartmuseum enabled", prometheus.GaugeValue, nil, nil)
 	allMetrics["system_notification_enable"] = newMetricInfo(instanceName, "system_notification_enable", "If notifications are enabled", prometheus.GaugeValue, nil, nil)
 	allMetrics["replication_latency"] = newMetricInfo(instanceName, "replication_latency", "Time in seconds to collect replication metrics", prometheus.GaugeValue, nil, nil)
+	allMetrics["image_vulnerability"] = newMetricInfo(instanceName, "image_vulnerability", "Get image vulnerability scan: None = 0, Low = 1, Medium = 2, High = 3 ", prometheus.GaugeValue, imageNames, nil)
```

Comment:
small nit, but looks like missing `Critical = 4` in the description?

Reply:
indeed, thanks for the hint. ```Critical = 4``` status is present now